README.md

```diff
@@ -7,188 +7,80 @@ apuX platform users and developers to customize firmware to their needs.
-cd sortbootorder
-# for mainline coreboot (4.5.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} make
-# for legacy coreboot (4.0.x)
-COREBOOT_ROOT=../coreboot-${BR_NAME} make distclean
-COREBOOT_ROOT=../coreboot-${BR_NAME} COREBOOT_REL=legacy make
-```
+Please note that you may have NFS server running on host what leads to ports
+conflicts.
 
-Building memtest86+
--------------------
+After starting NFS and HTTP you can boot apuX. Please enable network booting
+using [sortbootorder](https://github.com/pcengines/sortbootorder#theory-of-operation).
 
-```sh
-cd && cd coreboot-${BR_NAME}
-git clone https://github.com/pcengines/memtest86plus.git payloads/external/memtest86plus
-cd payloads/external/memtest86plus
-make -j$(nproc)
 ```
-
-cbfstool and adding/removing ROMs or payloads
----------------------------------------------
-
-`cbfstool` is result of `coreboot` build it gives ability to manipulate CBFS
-which filesystem for coreboot ROM.
-
-Usage examples:
-
-### Add iPXE ROM
-
-```sh
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom add -f payloads/external/ipxe/src/bin/8086157b.rom -n genroms/pxe.rom -t raw
+iPXE> ifconf net0
+iPXE> dhcp net0
+iPXE> chain http://<your_ip>:8000/menu.ipxe
 ```
 
-Above command add raw image (`8086157b.rom`, result of iPXE build) to
-`coreboot.rom` under the name `genroms/pxe.rom`. This makes SeaBIOS to auto
-detect iPXE ROM and execute it before entering menu.
-
-### Add sortbootorder
+Choose `Debian stable netboot 4.14.y` after boot login (`[root:debian]`) and
+run:
 
 ```
-# add sortbootorder
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom remove -n img/setup
-./build/cbfstool ./build/coreboot.rom add-payload -f payloads/pcengines/sortbootorder/sortbootorder.elf -n img/setup -t payload
+flashrom -p internal -w apuX_x.y.z.rom
 ```
 
-Above commands first remove already existing `img/setup` from CBFS and then add
-`sortbootorder.elf` as payload under the name `img/setup` to `coreboot.rom`.
-
+## Known issues
 
-### Add memtest86+
+### Incorrect privileges
 
 ```
-# add memtest86+
-cd && cd coreboot-${BR_NAME}
-./build/cbfstool ./build/coreboot.rom remove -n img/memtest
-./build/cbfstool ./build/coreboot.rom add-payload -f payloads/external/memtest86plus/memtest -n img/memtest - payload
+$ ./start.sh
+Error checking context: 'can't stat '/path/pxe-server/debian/debian-stable/var/lib/sudo''.         '
 ```
 
-Cross compilation in Docker container
----------------------------------------
+To fix that run:
 
-For advanced users and developers there maybe need to have development
-environment that is separated for working environment. Because of that you can
-use Docker containers as descried below.
-
-1. [Build container](docs/building_env.md)
-2. [Building firmware](docs/building_firmware.md)
-3. [Firmware flashing](docs/firmware_flashing.md)
+```
+sudo chown $USER:$USER . -R
```

Comment:
This is a bad approach. Changing owner of the system files will cause problems on the running system. Will be solved as proposed above.

Reply:
@miczyg1 maybe `.dockerignore` address that ?